drivers/adc/ad400x/ad400x.c

```diff
@@ -164,7 +164,7 @@ int32_t ad400x_spi_single_conversion(struct ad400x_dev *dev,
 	if (ret)
 		return ret;
 #endif
-	*adc_data = buf & 0xFFFFF;
+	*adc_data = buf & 0xFFFFFF;
```

Comment:
bit endianess i meant bit order.  ill changed it.  
the datasheet only specifies that 20 bits of data are sent MSB first. because i was always just seeing  16bits, i removed the mask and i confirmed that the higher nibble of the 3rd byte is the one that contains the LSB bits (which i do see when probing the spi bus).